README.md

```diff
@@ -0,0 +1,47 @@
+Create RPMs from a conda-gitenv
+===============================
+
+conda-rpms is designed to convert a resolved [conda-gitenv](https://github.com/SciTools/conda-gitenv) environment into a collection of RPM specs suitable for deployment on compatible platforms (Red Hat Enterprise Linux, Fedora, etc.).
+
+A high level description of the motivation for the creation of RPMs from a [conda-gitenv](https://github.com/SciTools/conda-gitenv) is described in the [centrally managed scientific software workflow](https://rawgit.com/pelson/conda-rpms/master/tmp_overview_docs/scitools-env-workflow.html) document.
+
+The code to do the conda distribution to RPM spec transformation itself is relatively simple, drawing on tools such as gitpython, Jinja2 and conda.
+
+Because a built RPM knows its destined installation location, a number of RPM abstractions have been made that enable us to retain conda's hard-linking and relocatability benefits.
+
+RPM Types
+=========
+
+Package RPM
+-----------
+
+Named in the form ``<RPM prefix>-pkg-<pkg name>-<pkg version>-<pkg build id>`` a package RPM represents the conda "package cache" (the thing that normally lives in `<root prefix>/.pkgs/<pkg name>-<pkg version>-<pkg build id>`). A package RPM *does not* express its dependencies and can not be usefully installed as a standalone entity.
+
+Tagged environment RPM
+----------------------
+
+Named in the form ``<RPM prefix>-env-<env name>-tag-<env tag>`` a tagged environment RPM represents a resolved conda environment.
+It depends on all Package RPMs that should be installed in order to produce a working environment. The tagged environment RPM knows it target installation prefix, and uses conda functionality at install time to link the Package RPMs to the desired installation prefix.
+
+Labelled environment RPM
+------------------------
+
+Named in the form `<RPM prefix>-env-<env name>-label-<label>`, a labelled environment is simply a meta package which depends on one, and only one, Tagged Environment RPM. Unlike the immutable tagged environment, a labelled environment will tend to change to reference newer tags as time moves on. This change is reflected in the labelled environment's `version`, meaning that it is only possible to ever have one labelled environment RPM installed per environment name.
+
+
+Usage
```

Comment:
I wonder whether this section should be above the RPM Types section. 

The RPM Types is detail, whereas the usage section is smaller and a bit more high level.

You don't necessarily need to understand what each of the RPMs are before you use it. You could always build your specs, then read more to figure out what each thing you've produced is.